src/index.js

```diff
@@ -76,13 +76,24 @@ module.exports = postcss.plugin(PLUGIN_NAME, (opts = {}) => {
 
   return async (css, result) => {
     const inputFile = css.source.input.file;
-    const resultPlugins = result.processor.plugins.filter(isResultPlugin);
     const pluginList = getDefaultPluginsList(opts, inputFile);
-    const plugins = [...pluginList, ...resultPlugins];
-    const loader = getLoader(opts, plugins);
+    let resultPluginIndex;
+    result.processor.plugins.some((plugin, i) => {
```

Comment:
Could you please use `findIndex` instead of `some`?. It can make the code shorter and cleaner

Reply:
yeh much better